governance/votes/Executive vote - May 06, 2024.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Template - [Executive Vote] Approve Coinbase Custody (RWA014-A) DAO Resolution - May 06, 2024
+summary: Approve Coinbase Custody (RWA014-A) DAO Resolution that extends the Coinbase Custody Rewards Program for six months.
+date: 2024-05-06T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Approve Coinbase Custody (RWA014-A) DAO Resolution - May 06, 2024
+
+The Governance Facilitators, dewix and Sidestream have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
```

Comment:
```suggestion
The Governance Facilitators, Dewiz, and Sidestream have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
```

Reply:
Good catch!
@blimpa try to copy and paste names and variables to avoid this kind of mistakes